### PR TITLE
14356 - Fix cheque date. Revert previous changes. Put in split for DateTime.

### PR DESCRIPTION
**Pull request by seeker25**
[#14356](https://github.com/bcgov/entity/issues/14356)

### CODE REVIEW

File: src/components/ReviewRoutingSlip/ReviewRoutingSlipChequePayment.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
         label="Cheque Date"
         persistent-hint
         hide-details
-        :value="payment.paymentDate ? formatDisplayDate(payment.paymentDate, 'MMMM DD, YYYY') : '-'"
+        :value="payment.paymentDate ? formatDisplayDate(payment.paymentDate.split('T')[0], 'MMMM DD, YYYY') : '-'"
```
Comment (MatthewCai2002) [inline src/components/ReviewRoutingSlip/ReviewRoutingSlipChequePayment.vue:23]: ~~Not really sure how splitting the paymentDate between the date and time is causing it to show the correct date.~~
Oh wait removing time from the date means that when we convert paymentDate into a moment object it doesn't get shot back 8 hours into the previous day ahhh

**Review by MatthewCai2002 — APPROVED** [review]
looks good to me

File: src/util/common-util.ts [hunk 2]
```diff
@@ -6,16 +6,12 @@ import { Address, BaseAddressModel } from '@/models/Address'
...
 
 export default class CommonUtils {
   // Formatting date in the desired format for displaying in the template
   static formatDisplayDate (date: Date, format?: string) {
-    if (date) {
-      return moment.tz(date, 'Canada/Vancouver').format(format || 'MMM DD, YYYY')
-    } else {
-      return ''
-    }
+    return date ? moment(date).format(format || 'MMM DD, YYYY') : ''
```
Comment (kialj876) [inline src/util/common-util.ts:14]: Shouldn't you set the timezone here? Wont all the dates be in UTC?

Comment (seeker25) [reply src/util/common-util.ts:14]: Probably, but I just focused on what was in the ticket (The Cheque date). Changing this impacts quite a few of the dates.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**